### PR TITLE
Update information of 2020.1 meetings

### PR DESCRIPTION
**Pull request by AndrewIjano**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -39,9 +39,9 @@ <h2>Novos hacks toda semana</h2>
         {% include next-event-table.html %}
     </div>
     <div class="home--section home--color-scheme-5">
-            <div class="home--content home--right-text">
-                <h2>Quartas-feiras,<br> às 13h15</h2>
-                <p>No Auditório do CCSL, no IME-USP.</p>
+            <div class="home--content home--color-scheme-5">
```
Comment (pedro823) [inline src/index.html:42]: doesn't the upper div already contain this class?

File: src/_includes/telegram-logo.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg id="Bold" enable-background="new 0 0 24 24" height="512" viewBox="0 0 24 24" width="512" xmlns="http://www.w3.org/2000/svg"><path d="m12 24c6.629 0 12-5.371 12-12s-5.371-12-12-12-12 5.371-12 12 5.371 12 12 12zm-6.509-12.26 11.57-4.461c.537-.194 1.006.131.832.943l.001-.001-1.97 9.281c-.146.658-.537.818-1.084.508l-3-2.211-1.447 1.394c-.16.16-.295.295-.605.295l.213-3.053 5.56-5.023c.242-.213-.054-.333-.373-.121l-6.871 4.326-2.962-.924c-.643-.204-.657-.643.136-.953z"/></svg>
```
Comment (pedro823) [inline src/_includes/telegram-logo.svg:1]: where do we use this? is this necessary in this PR?

File: src/_sass/home.scss [hunk 3]
```diff
@@ -92,6 +92,15 @@
     * {
         color: #FFF;
     }
+    .home--button {
+        background-color: #FFF;
```
Comment (pedro823) [inline src/_sass/home.scss:96]: Don't we have a var for this color? if not, we should create it

File: src/index.html [hunk 4]
```diff
@@ -39,9 +39,9 @@ <h2>Novos hacks toda semana</h2>
...
+                <h2>Sábados,<br> às 13h</h2>
```
Comment (pedro823) [inline src/index.html:43]: *nit*: close this `<br />`.

File: src/index.html (hunk 1 above)
Comment (AndrewIjano) [reply src/index.html:42]: Yes, it does... I don't know why I thought it was necessary

File: src/_includes/telegram-logo.svg (hunk 2 above)
Comment (AndrewIjano) [reply src/_includes/telegram-logo.svg:1]: I don't even know why this is there. Sorry. I'll remove it

File: src/_sass/home.scss (hunk 3 above)
Comment (AndrewIjano) [reply src/_sass/home.scss:96]: Hm, are you asking for vars for "white-lh" and "black-lh"? I can do this in another PR

File: src/index.html (hunk 4 above)
Comment (AndrewIjano) [reply src/index.html:43]: Ok!

**Pull request merged**